src/Symfony/Component/Console/Descriptor/TextDescriptor.php

```diff
@@ -113,9 +113,12 @@ protected function describeInputDefinition(InputDefinition $definition, array $o
      */
     protected function describeCommand(Command $command, array $options = array())
     {
+        $messages = array('<comment>Description:</comment>', ' '.$command->getDescription(), '');
+
         $command->getSynopsis();
         $command->mergeApplicationDefinition(false);
-        $messages = array('<comment>Usage:</comment>', ' '.$command->getSynopsis(), '');
+        $messages[] = '<comment>Usage:</comment>';
+        $messages[] = ' '.str_replace("\n", "\n ", $command->getSynopsis())."\n";
```

Comment:
@stof i used here the same str_replace strategy as used elsewhere below. I take that the synopsis can have break lines and this should take care of the alignment? is this correct? Also I am not sure about the $command->getDescription() above if also the str_replace is needed there in case the description is also lengthy. Please let me know.

Besides this I will deal with the tests...